### ISSUE TITLE
Integrity checking

### ISSUE THREAD

**Issue by gamb** [purcell/postgresql-migrations#2]
Thinking it might be useful if `apply_migration()` checks that the ddl in migrations.sql matches the applied table?

This would help to catch modification after the fact or unintentionally reused identifiers, which can silently skip at the moment.

**Comment by purcell** [comment]
Agreed, the reused identifiers thing is a definite problem. I'll probably make a change along these lines in due course.
